libs/blocks/caas/caas.css

```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
   }
 }
 /* End Dexter Modal */
+
+/* CaaS Overrrides */
+.tool-content {
+    grid-template-columns: 340px 20px 1fr;
+    gap: 6px;
+}
+
+dt.title span {
+  font-size: 18px;
+}
+
+dt.title.is-expanded span, dt.title.is-expanded button {
+  font-weight: 300;
+}
+
+.multifield {
+  background-color: #404040;
+  border-bottom: 0;
+  padding: 0;
+  margin: 14px 0 0;
+}
+
+.multifield > .multifield-header h3 {
+  font-size: 18px;
+  font-weight: 700;
+  margin: .5rem;
+  margin: 2px 6px;
+}
+
+.multifield > .multifield-set > .multifield-fields .fields {
+  margin: 0;
+}
+
+.accordion {
+  margin: 0 0 0 6px;
+}
+
+dd.content.is-expanded {
+  margin-top: 0;
+}
+
+dd.content .field {
+  margin: 8px 0;
+}
+
+.accordion-item {
+  margin: 1px 0;
+}
+
+.accordion > .accordion-item {
+    margin: 1px 0;
```

Comment:
Extra indentation; rule is identical t the previous, they can be grouped together